lib/measured/unit.rb

```diff
@@ -23,10 +23,11 @@ def inspect
 
   def <=>(other)
     if self.class == other.class
-      if other.names != @names
-        other.names <=> @names
+      name_cmp = @names <=> other.names
```

Comment:
Was following the standard library format (e.g., `String#casecmp`), but yeah, I'm all for full words 👍